src/Message/MerchantUrlsDataTrait.php

```diff
@@ -33,12 +33,14 @@ public function getMerchantUrls(): array
     {
         $this->validate('notifyUrl', 'returnUrl', 'termsUrl', 'validationUrl');
 
+        $returnUrl = $this->getReturnUrl();
+
         $merchantUrls = [
-            'checkout' => $this->getReturnUrl(),
-            'confirmation' => $this->getReturnUrl(),
-            'push' => $this->getNotifyUrl(),
-            'terms' => $this->getTermsUrl(),
-            'validation' => $this->getValidationUrl(),
+            'checkout'     => $returnUrl,
```

Comment:
please do not change this indenting

Reply:
Sorry, stupid config in my linter. Reverted indentation in latest commit.